src/NextjsServer.ts

```diff
@@ -117,6 +117,7 @@ export class NextjsServer extends Construct {
       ...getCommonFunctionProps(this),
       code: Code.fromBucket(asset.bucket, asset.s3ObjectKey),
       handler: 'index.handler',
+      description: 'Next.js server handler',
```

Comment:
Great idea!

Reply:
Should we do this for all functions?

Reply:
I think most of the functions we're explicitly creating have descriptions now. Some of the ancillary functions do not, but to me that seems "fine" given most of the time when I'm hunting for a function to debug it's either server handler (99% of the time), the queue handler, or the image handler, and those are now all easily identifiable from the description. 

<img width="1148" alt="Screenshot 2023-10-12 at 9 27 41 AM" src="https://github.com/jetbridge/cdk-nextjs/assets/21270844/003d3aa6-9a39-41a1-a55c-df6b2f8e1324">